### ISSUE TITLE
Grammar file is no longer valid XML

### ISSUE THREAD

**Issue by lildude** [mathworks/MATLAB-Language-grammar#32]
👋  Hi there. I'm the lead maintainer of https://github.com/github/linguist, the library used for syntax highlighting on GitHub.com.  We use this grammar for highlighting MATLAB files on GitHub.com. I'm in the process of making a new release of Linguist which involves updating all the grammars and our compiler has failed because the grammar file is no longer valid XML:

```
$ xmllint vendor/grammars/MATLAB-Language-grammar/Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage                             lildood@gmail.com
vendor/grammars/MATLAB-Language-grammar/Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage:1233: parser error : StartTag: invalid element name
			<string>(\)|(?<!\.\.\.).\n)</string>
			               ^
vendor/grammars/MATLAB-Language-grammar/Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage:1338: parser error : StartTag: invalid element name
			<string>(\}|(?<!\.\.\.).\n)</string>
			               ^
```

https://github.com/mathworks/MATLAB-Language-grammar/pull/28 appears to be the PR that broke things.

Looks like those two `<` chars need escaping or removing.

**Comment by rlivings39** [comment]
Hi @lildude this should be fixed by #31 

@gauravsavadi can you please look at merging that PR?

**Comment by rlivings39** [comment]
Fixed by #31

**Comment by lildude** [comment]
Great! This missed today's update but will make it into the next.
